src/components/AddCharacterForm.tsx

```diff
@@ -0,0 +1,61 @@
+import {
+  Modal,
+  ModalOverlay,
+  ModalContent,
+  ModalHeader,
+  useDisclosure,
+  VStack,
+  Input,
+  FormControl,
+  HStack,
+  WrapItem,
+  Button,
+  ModalCloseButton,
+} from '@chakra-ui/react';
+import ButtonComp from './Button';
+
+const AddCharacterForm = () => {
+  const { isOpen, onOpen, onClose } = useDisclosure();
+
+  return (
+    <>
+      <ButtonComp onClick={onOpen} buttonText="Add a character" />
+      <Modal isOpen={isOpen} onClose={onClose}>
+        <ModalOverlay />
+        <ModalContent bg={'gray.800'}>
+          <ModalHeader textColor={'white'}>Add a character </ModalHeader>
+          <ModalCloseButton color={'white'} />
+          <VStack p={4} spacing={4}>
+            <FormControl>
+              <Input placeholder="Name" textColor={'white'} />
```

Comment:
It'd be great if you could look into styling the `Input` components a bit more so that they look more like the Figma! (Some changes you could make: set border radius to 0, background color to a shade of grey lighter than the modal, border to 0, size to large, etc.)

Also notice that each `Input` has the same styles and that the only difference between them is the placeholder. That's a good indication that it'd probably be better to make a custom `Input` component! This custom `Input` component should live in `src/components` & take in a placeholder prop---the process should be pretty similar to what y'all did with the `Button` component (you'd then import the custom `Input` component you made & use it instead of Chakra UI's default `Input`).